.github/workflows/integration-test-review.yml

```diff
@@ -80,15 +83,15 @@ jobs:
             const { owner, repo } = context.repo;
 
             // Read the PR number from the downloaded and unzipped artifact.
-            const issue_number = Number(fs.readFileSync(path.join(temp, 'pr_number')));
+            const pr_number = Number(fs.readFileSync(path.join(temp, 'pr_number')));
 
             // Get the URL of the PR so we can add a link in the PR comment.
-            const { html_url } = await github.rest.issues.get({ owner, repo, issue_number });
+            const { html_url } = await github.rest.pulls.get({ owner, repo, pr_number });
```

Comment:
Interesting that the REST API cares about issues vs PRs; the web interface treats them the same, i.e. `.../issues/1014` would redirect here. :shrug:

Reply:
Yes, it's rather confusing. I noticed the redirection too, but for some reason, the value of `html_url` was coming back as `undefined`, thus causing the broken link. I _think_ that using `pulls` instead of `issues` should give us back the expected URL.

Reply:
Awesome job getting the workflow working ;)